packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -122,20 +122,36 @@ abstract class RenderSliverBoxChildManager {
   bool debugAssertChildListLocked() => true;
 }
 
+/// Parent data structure used by [RenderSliverMultiKeepAliveBoxAdaptor].
+class SliverMultiKeepAliveBoxAdaptorParentData extends SliverLogicalParentData  {
```

Comment:
also this should probably be abstract

Reply:
Yes I wasn't very proud of "SliverMultiKeepAliveBoxAdaptorParentData" :confused:, "KeepAliveParentData" is much better.
I'm not very familiar with mixins. Why do you think it should be a mixin instead of a superclass?

Reply:
Mostly because of how orthogonal the concepts are. There's nothing in the keep-alive stuff that cares about the child model, so it's hard to justify introducing the opinion about the child model in just the class name.